### ISSUE TITLE
Debug using a selected environment -Feature Request / Idea

### ISSUE THREAD

**Issue by ConX-Ryan** [Azure/service-fabric-issues#884]
Great SDK love it keep up the good work!

Can I please propose a feature where by i can select the application environment / application parameters associated when debugging? that would be super handy I think.

**Comment by MikkelHegn** [comment]
Hi @ConX-Ryan - thank you for the propsal. I'll hand it over to the Visual Studio tooling people. There is an option to run any script after application deployment, to create services with specific parameters. Maybe that could be used in this scenario?
@dbreshears - to help with this feature.

**Comment by dbreshears** [comment]
@ConX-Ryan , Yep, thanks, as you discovered we pass in either Local.1Node.xml or Local.5Node.xml of the publish profile to the deploy script on F5 based on how the cluster is configured   I am not sure we have a good solution for you at this time and will add this suggestion to the tooling backlog.  I believe the feature Mikkel is referring too is to create a script named "Start-Service.ps1" in the Scripts folder  and we will invoke it on F5 after deployment.   It seems that is going to be too late to solve this problem though as we get the parameters file from the publish profile that is passed into the deploy script which is what you are wanting to choose.   The only option I see  is to modify the Deploy script in your solution to change each time your want to use different  publish profile/parameters,    but at that point you might as well just change the parameters for Local.1Node or Local.5Node?   Changing that also would affect direct publish from VS.

**Comment by olitomlinson** [comment]
I'm experiencing the same blocker. I need to be able to override the publish profile when using F5 debugging.

We have two dev teams contributing to the same Service Fabric solution. One team is making adjustments in light of running the solution on prem. Another team is making adjustments in light of running the solution in Azure.

Each team requires different settings through the config like so : 

Local.TeamA.1node.xml
Local.TeamA.5node.xml
Local.Team**B**.1node.xml
Local.Team**B**.5node.xml
etc

Please please please give us a way to solve this!
